### PR TITLE
[CERT-9956] GH app options for reviews

### PR DESCRIPTION
**Pull request by H00N24**
## 🚀 Pull Request Overview

In this PR we're adding:

- Option to name the reviews.
- Option to hide successful jobs in the review.
- Option to hide reviews entirely.
---

### 📜 Tests Checklist

Before submitting this PR, ensure that all tests pass **and** meet the following conditions:

| Category                  | Status                          | Compilation Comment | Results Review |
|---------------------------|----------------------------------|----------------------|----------------|
| **fail_to_start**         | ❌ Failed with compilation error | 🟢 Yes               | 🔴 No          |
| **violated_rules**        | ✅ Passed, with violations found | 🔴 No                | 🟢 Yes         |
| **verified_rules**        | ✅ Passed without violations     | 🔴 No                | 🟢 Yes         |
| **solana_violated_rules** | ✅ Passed, with violations found | 🔴 No                | 🟢 Yes         |
| **sui_verified_rules**    | ✅ Passed without violations     | 🔴 No                | 🟢 Yes         |

> Please verify that your changes meet the above conditions for each category of tests.  
> If something doesn't match, investigate before submitting the PR.

---

### 📋 Additional Notes (Optional)

- Any risks, edge cases, or implementation notes for reviewers:

### CODE REVIEW

File: scripts/gh-app-integration.sh [hunk 1]
```diff
@@ -57,7 +60,10 @@ PAYLOAD=$(jq -n \
   --arg config_hash "$CERTORA_CONFIGURATIONS_HASH" \
   --argjson replace_comments "$CERTORA_REPLACE_COMMENTS" \
   --argjson pr_number "${PR_NUMBER:-null}" \
-  '{group_id: $group_id, commit: $commit, action_ref: $action_ref, config_hash: $config_hash, replace_comments: $replace_comments, pr_number: $pr_number}')
+  --arg job_name "$CERTORA_JOB_NAME" \
```
Comment (fnast9) [inline scripts/gh-app-integration.sh:63]: Could you please clarify on how the user set the value? Is it an env var?

**Review by fnast9 — APPROVED** [review]

**Review by yoav-el-certora — COMMENTED** [review]
I think we should also update the README with the new instructions.
